packages/mantine/src/components/table/__tests__/TableFilter.spec.tsx

```diff
@@ -97,6 +99,29 @@ describe('Table.Filter', () => {
         expect(screen.getByRole('textbox')).toHaveValue('');
     });
 
+    it('clear the filter if the global state filter is cleared', async () => {
+        const user = userEvent.setup({delay: null});
+
+        const Fixture = () => {
+            const {clearFilters} = useTable();
+            return <Button data-testId="clear-button" onClick={clearFilters} />;
+        };
+
+        await render(
```

Comment:
`render` doesn't return a promise if I'm not mistaken

```suggestion
        render(
```